tests/BasicSearchTests.FunctionalTests.Core/V3SearchFunctionalTests.cs

```diff
@@ -37,5 +37,23 @@ public class V3SearchFunctionalTests : BaseFunctionalTests
             Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
             Assert.NotNull(result.Data);
         }
+
+        [Fact]
+        public async Task ResultForJsonNetHasDownloads()
+        {
+            // Act
+            var response = await Client.GetAsync(new V3SearchBuilder() { Query = "JSON.Net" }.RequestUri);
+            var result = await response.Content.ReadAsAsync<V3SearchResult>();
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
```

Comment:
I don't see how having a helper function (`AssertSearchResultHasData`) harms the structure of the test case--it's fairly explicit about what it's asserting and if we decide we want to change what each test is asserting specifically we can just split the function. Ultimately however, I'm fine with how it is now.